packages/web/playwright.config.ts

```diff
@@ -101,8 +101,11 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'npm run start:spa:stage',
-    url: 'http://localhost:3001',
+    // On CI the build already exists, so we don't need to use the turbofied `preview:stage` command
+    command: process.env.CI
+      ? 'npm run preview:stage:ci'
+      : 'npm run preview:stage',
```

Comment:
so we dont want to run the start:spa:stage for local either so we can get more consistent testing locally also?

Reply:
Yes, I wanted the local testing to be consistent with CI testing. The downside is that if we are working on tests locally, and need to make changes to the app we will need to run `npm run build:stage`